File: stone-game/stone-game.cpp
class Solution {
public:
    bool stoneGame(vector<int>& p) {
        int a = 0 , b = 0 ;
        sort(p.begin(),p.end());
        reverse(p.begin(),p.end());
        for(int i = 0 ; i < p.size() ;i++){
            if(i%2==0)
            a += p[i];
            else
            b += p[i];
        }
        if(a>=b)
        return true;
        else
        return false;
        
    }
};

File: stone-game/stone-game.py
class Solution:
    def stoneGame(self, piles: List[int]) -> bool:
        a ,  b = 0 , 0 
        piles = sorted(piles)
        piles = piles[::-1]
        for i in range(len(piles)):
            if i%2==0:
                a += piles[i]
            else:
                b += piles[i]
        if a>=b:
            return True
        else:
            return False